src/dependency-core/Core/ObsoleteMessage/ObsoleteMessage.cs

```diff
@@ -0,0 +1,27 @@
+#nullable enable
+
+namespace PrimeFuncPack
+{
+    internal static class ObsoleteMessage
```

Comment:
1. Would you consider to shorten the messages to `"This method is obsolete. Call %NewMethodName% instead."` by avoiding specifying a class name? (because the new methods are declared in the same classes)
2. Let's change `Use` to `Call` to match the convention already used in both .NET and PFPack for obsolete methods.

Reply:
Done